packages/utils/src/async.ts

```diff
@@ -0,0 +1,26 @@
+//
+// Asynchronous functionality
+// ---
+import { safeSetTimeout } from './error';
+
+// tslint:disable-next-line:ban-types
+export function throttle(callback: Function, timeout: number) {
```

Comment:
Need to check whether we can make this more specific than Function?

Reply:
It looks like `lodash` has a generic `Function` as well: https://github.com/lodash/lodash/blob/master/throttle.js (though I'm not sure the jsdoc is used for the type checking, I noted that the [throttle type at DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/lodash-es/throttle.d.ts) just re-exports)